sinonjs/samsam
# Verify assertions

#### Purpose (TL;DR) - mandatory

With this PR, each test case is required to have at least one referee assertion.

#### Solution  - optional

I was experimenting with the new [`referee.verifier()`](https://sinonjs.github.io/referee/#refereeverifier) utility and Mocha for a while now and found the `--file` option with a global helper to be quite elegant. We don't need to change any of the test cases.

If we're happy with this, we can apply the same to the Sinon project. We could also make the `verifier.js` fixure a new package maybe.

#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm t` and see that all tests are green
4. Comment out a random assertion
5. `npm t` and see that all tests are green except the one without assertions

#### Checklist for author
